installer/scripts/omsadmin.sh

```diff
@@ -346,6 +362,28 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
Spoke with @keikhara ; our aim is to make the resource limits as specific as possible. If we place the limit at the group level instead of the user level, the omsagent could be pointlessly throttled when the nxautomation user misbehaves.

I will send out an email asking the automation team to add a process limit for the nxautomation worker, after they have performed similar testing for their user's processes.

Given this, please approve this PR.